client/components/session/session.service.js

```diff
@@ -0,0 +1,49 @@
+class SessionService {
+    constructor ($q, OvhApiMe, OvhApiProducts) {
+        this.$q = $q;
+        this.OvhApiMe = OvhApiMe;
+        this.OvhApiProducts = OvhApiProducts;
+    }
+
+    getUser (force) {
+        // Check if cached
+        if (!!this.user) {
+            return $q.when(this.user);
+        }
+
+        // Check if already deferred
+        if (!!this.userDeferred && !force) {
+            return this.userDeferred.promise;
+        }
+
+        // Get User
+        this.userDeferred = this.$q.defer();
+        this.OvhApiMe.Lexi()
+            .get().$promise
+            .then((result) => this.userDeferred.resolve(result));
+
+        return this.userDeferred.promise;
+    }
+
+    getProducts (force) {
```

Comment:
There is already a "ProductsService" (https://github.com/ovh-ux/ovh-manager-cloud/blob/master/client/components/products/products.service.js)